### DISCUSSION TITLE
Why the `JOIN BETA CHANNEL` button is in supervisor section?

### DISCUSSION THREAD

**Discussion by Misiu** [home-assistant/frontend#10438] [Configuration] [answered]
I wanted to join the beta channel to test all the changes coming to frontend, but I noticed that `JOIN BETA CHANNEL` is located in the `Supervisor` group:

![image](https://user-images.githubusercontent.com/1741838/139230152-31a616ce-e0f3-4baf-9b50-a3c60db7cefb.png)

and when you click it you see this dialog:
![image](https://user-images.githubusercontent.com/1741838/139230211-1ec0fdb4-f311-4677-bec4-4f19a277f217.png)

so you are joining the beta channel for Core, Supervisor, and OS.

I think it is confusing and should be moved to another place, but I'm not sure what would be the ideal place.
Any thoughts?

**Comment by ludeeus** [comment, marked as answer]
This is in the correct place, this is a feature of the supervisor.
This is setting the channel for the supervisor.
The channel is used to determine which version file to read updates from.
- [stable](https://version.home-assistant.io/stable.json)
- [beta](https://version.home-assistant.io/beta.json)
- [dev](https://version.home-assistant.io/dev.json)
